### PR TITLE
Lots more books

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary
```SUMMARY: Content "lots more books"```
#### Purpose of change

This request 
- adds a number of new books and book abstracts, 
- reorganizes book/misc.json, 
- adds book/spiritual.json and moves those books from misc.json, 
- and adds new content to loot tables as appropriate.

#### Describe the solution

Uh, in no particular order:

1. add &quot;copy-from&quot;: &quot;paperback_novel&quot; to novel_satire and novel_crime (and made associated adj)
2. correct an accidental change to novel_horror (from #37166)
3. add novel_crime_midnight, and three new swashbuckling books
5. add a few more fairy snippets
6. add paperback_occult abstract
7. add occult_economicon, occult_bobliographon, and occult_glimpses
8. add three romance paperback abstracts, each representing a publisher of romance novels
9. ten new romance novels
10. add a western abstract and six books copying from it
12. add holybook abstract and associate holy books with it
13. move holybook abstract and associated holy books into their own file: spiritual.json
14. add a dodge abstract and new mid-level dodge book
15. added 5 new satires: The House of God, Catch-22, Master and Margarita, Cat's Cradle, and a Handful of Dust
16. create tables for magazines including SUS_book_nonf_zine
17. add hard back archery abstract
18. add 8 "homemaking" books in a file called nonfiction.json, where further nonfiction clutter will go. Add additional nonfiction clutter to same file, including a number of books about psychology that I hope can add some creepiness to labs and mansions.
19. add a new speech book _"Nail Your Law Firm Interview"_
20. move some kids books from misc to a new file young.json, and add a number of young adult books (fiction clutter) to same file
21. add romances and westerns to SUS_fiction_bookcase in SUS/library.json
22. add a "cascading" set of tables to SUS/library.json, each returning a single book with greater degrees of specificity
23. add 4 new pulp fiction novels
24. move abstracts to abstract.json

##### occult books added to misc

I added two &quot;occult paperbacks&quot; on the Church of Subgenius. Church of Subgenius is an existing entity within CDDA, and the books are loyal to that faith tradition, meaning that they are inherently nonsensical in nature. 

The third paperback references *The Church of Starry Wisdom*, and I didn't want to compromise existing lore; so this book is purposefully portrayed as an unreliable source of info.

##### added new file spiritual.json

*spiritual.json* could perhaps be better named *holy.json* or *inspirational.json*. The former due to the subject matter, naming convention, and presumably intent. The later due to the flag. "Spiritual" is useful because of the trait.

##### OC vs IRL

I added fiction books that were of my own made creations, and I also added famous real novels.  The file already included dozens (hundreds?) of titles of real novels.  Do we have a preference for real world content vs OC?  Is having a mix of both problematic?

##### Organization

I organized content first as fiction or nonfiction, and then by type (paperback or hardback mostly, but there are others printed materials too).

This was only semi-intentional, and stems from the convention that skill training books tended to be nonfiction and hard backs, whereas non-skill books (ie., books for fun, clutter books, books for lore) tended to be fiction and have physical stats suggesting paperback.  Regardless of the merits of one taxonomy decision vs another, it is better to have content organized than not.  I'm not wedded to my scheme, but I do wish to continue to organize this content so that it is convenient.

I hope to add at least a dozen hardback fiction books for variety.  These will take up significantly more inventory space.  However this PR does not include that.  Additionally, I hope to add dozens of nonfiction titles that are not skill books. This PR does not include those.

The homemaking category might be a tad sexist. These are books about gardening, beauty, entertaining, and home decor. It isn't visible in game however, and I don't want to further complicate this request.

##### library.json

The extensive "cascading" set of tables added to SUS/library.json violates the letter of that project, but conforms to the spirit of it.  The single tables are not meant to reflect a specific piece of furniture, and are meant to facilitate that admirable goal.

##### misc
Some things I wanted to do but resisted the impulse:

* Dungeon Masters Guide has a bashing of 7, more than a metal pot, and over twice as deadly as a steel ladle or a can opener. I considered bringing it down to at least 6.
* Considered change _priest_diary_ from fun: -1 to fun: 1
* Considered change read time of _story_book_ - it currently has 28 chapters and each one takes nearly an hour to read; what kind of short stories are these?

#### Testing

* I used a relative on novel_satire and so I tested that price was in fact 850 (750+100)
* used relative on novel_swash_searovers and tested that weight was 557 (1.2lbs) and there were 24 quickly read chapters of fun 2 each
* check that novel_swash_blkfreight had fun 4 and only 8 chapters
* ensure game loads
* use "Test Item Group" debug tool to test item spawns, particularly SUS_fiction_bookcase, which favors fiction genres that have not been updated
* double check that item_groups and abstracts allow for nested relationships more than one deep
* Holy books all draw from one abstract with two materials: paper and cardboard. Test how this impacts scrap/deconstruct/butcher.

##### TODO

* Cat's Cradle (_book_fict_soft_satire_cats_) has 127 chapters. Each one should take 1m to read (maybe 2m?) and provide 1 fun (because there is nothing lower)
* test that fantasy novels now look (merely have the tile) like dungeon master guide
* test that skill: foo overrides chapter setting

#### Additional context

I tangled up my master branch, so there are some unrelated files here.  It doesn't look like anything problematic and it is unintentional.

&quot;Buzzards Bay&quot; with no apostrophe is not a typo; BB is off the coast of Rhode Island and Mass and I triple checked the spelling.

I could not get the time parameter to function correctly as a relative value. A guess is this might be a fault of the abstract uses a string and my instance used an integer? I side stepped this by just taking it out of the relative (defining it explicitly).

While I was looking into giving occult books INSPIRATIONAL flags I discovered that Spiritual is multiplicative, but nearly every book flagged has fun 1 so that is irrelevant. The _priest_diary_ has fun: -1, meaning that a Spiritual character will find priest diary three times as boring as a non Spiritual character - clearly a bug. This should be addressed, but is outside the ever-expanding scope of this PR.

##### Future plans

Once I've added a greater variety of books, I intend to take a look at book stores and libraries to see if I can improve those locations with both more and better organized books.

### CODE REVIEW

**Comment by Jerimee** [comment]
@I-am-Erk will you please take a look at how I added romance novels to SUS and let me know if I should add my content to SUS (or SUS-like) tables differently?

You'll see I changed romance in SUS_fiction_bookcase to no longer spawn 1 to 5 copies, but instead to have a 40% each of spawning one copy of a specifically titled romance book.

So it will no longer be possible to get four of novel_romance the way one might get four of novel_satire, instead one might get paperback_romance_conq, paperback_romance_dd, and a single instance of novel_romance

_(Clearly these books are, by design, extremely similar. The difference are in the titles and descriptions - primarily for flavor/clutter/variety.)_

File: data/json/items/book/archery.json [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
-    "material": [ "paper" ],
+    "price": 1900,
+    "price_postapoc": 5000,
+    "material": [ "paper", "cardboard" ],
     "symbol": "?",
+    "looks_like": "story_book",
     "color": "green",
+    "intelligence": 7,
     "skill": "archery",
+    "time": "20"
```
Comment (ZhilkinSerg) [inline data/json/items/book/archery.json:17]:
```suggestion
    "time": "20 m"
```

File: data/json/items/book/dodge.json [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+    "price": 1000,
+    "price_postapoc": 5000,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "looks_like": "story_book",
+    "color": "light_blue",
+    "flags": [ "TINDER", "FLAMMABLE" ],
+    "skill": "dodge",
+    "intelligence": 4,
+    "time": 10,
```
Comment (ZhilkinSerg) [inline data/json/items/book/dodge.json:18]:
```suggestion
    "time": "10 m",
```

File: data/json/items/book/misc.json [hunk 3]
```diff
@@ -14,10 +14,161 @@
     "color": "light_cyan",
     "flags": [ "TINDER", "FLAMMABLE" ],
     "intelligence": 4,
-    "time": "15 m",
+    "time": 15,
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:17]:
```suggestion
    "time": "15 m",
```

File: data/json/items/book/misc.json [hunk 4]
```diff
@@ -14,10 +14,161 @@
...
+  {
+    "abstract": "book_nonf_tpl",
+    "type": "BOOK",
+    "name": "Generic Nonfiction Book",
+    "description": "template for a manuscript purporting to be factual",
+    "intelligence": 5,
+    "symbol": "?",
+    "looks_like": "story_book",
+    "color": "light_blue",
+    "time": 20
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:30]:
```suggestion
    "time": "20 m"
```

File: data/json/items/book/misc.json [hunk 5]
```diff
@@ -14,10 +14,161 @@
...
+    "volume": "700 ml",
+    "price": 750,
+    "price_postapoc": 2900,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "looks_like": "story_book",
+    "color": "dark_gray",
+    "flags": [ "TINDER", "FLAMMABLE" ],
+    "intelligence": 6,
+    "time": 35,
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:84]:
```suggestion
    "time": "35 m",
```

File: data/json/items/book/misc.json [hunk 6]
```diff
@@ -14,10 +14,161 @@
...
+  {
+    "abstract": "paperback_romance_spb",
+    "type": "BOOK",
+    "name": "Sweet Providence Romance Novel",
+    "description": "Sweet Providence Books is a publisher of discount romance paperbacks easily recognized by their blue and yellow cover illustrations. Despite the adult nature of their subject matter, the books tend to be under 250 pages of large print written in a vocabulary consistent with a 4th grade reading level.",
+    "//": "these are books are tiny",
+    "weight": "200 g",
+    "volume": "400 ml",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price": -351, "price_post": -800, "time": -4, "chapters": -8, "intelligence": -1 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:98]:
```suggestion
    "relative": { "price": -351, "price_post": -800, "time": "-4 m", "chapters": -8, "intelligence": -1 }
```

File: data/json/items/book/misc.json [hunk 7]
```diff
@@ -14,10 +14,161 @@
...
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price_post": -400, "chapters": 9 }
+  },
+  {
+    "abstract": "book_fict_soft_satire_tpl",
+    "type": "BOOK",
+    "name": "Satire Template",
+    "description": "template for mass produced satirical fiction",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "intelligence": 3, "price": 249, "time": 5 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:146]:
```suggestion
    "relative": { "intelligence": 3, "price": 249, "time": "5 m" }
```

File: data/json/items/book/misc.json [hunk 8]
```diff
@@ -14,10 +14,161 @@
...
+    "description": "template for magazine",
+    "intelligence": 4,
+    "weight": "60 g",
+    "volume": "250 ml",
+    "price": 400,
+    "price_postapoc": 100,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "pink",
+    "time": 7,
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:161]:
```suggestion
    "time": "7 m",
```

File: data/json/items/book/misc.json [hunk 9]
```diff
@@ -14,10 +14,161 @@
...
+    "fun": 1
+  },
+  {
+    "abstract": "book_nonf_zine_news_tpl",
+    "type": "BOOK",
+    "name": "News Magazine Template",
+    "description": "template for news magazine",
+    "copy-from": "book_nonf_zine",
+    "relative": { "intelligence": 1, "time": 1 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:170]:
```suggestion
    "relative": { "intelligence": 1, "time": "1 m" }
```

File: data/json/items/book/misc.json [hunk 10]
```diff
@@ -746,14 +559,22 @@
...
-    "copy-from": "paperback_novel"
+    "copy-from": "book_fict_soft_tpl"
+  },
+  {
+    "id": "novel_crime_midnight",
+    "type": "BOOK",
+    "name": { "str": "Midnight Cop", "str_pl": "copies of Midnight Cop" },
+    "description": "In this bare-knuckled potboiler, a ruthless police detective schemes to shut down local crime lords by pitting them against each other.  When long-simmering resentments finally flare the city learns why they call it \"the dead of night.\"",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "time": 5, "chapters": -8 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:570]:
```suggestion
    "relative": { "time": "5 m", "chapters": -8 }
```

File: data/json/items/book/misc.json [hunk 11]
```diff
@@ -844,7 +673,81 @@
...
-    "copy-from": "paperback_novel"
+    "copy-from": "book_fict_soft_tpl"
+  },
+  {
+    "id": "paperback_romance_circuses",
+    "type": "BOOK",
+    "name": "Love and Circuses",
+    "description": "The passionate saga of two Boston politicians fiercely battling each other for the mayor's office, and for Lydia's hand in marriage.",
+    "copy-from": "paperback_romance_spb",
+    "relative": { "chapters": 2, "time": 3, "weight": 111 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:684]:
```suggestion
    "relative": { "chapters": 2, "time": "3 m", "weight": 111 }
```

File: data/json/items/book/misc.json [hunk 12]
```diff
@@ -867,18 +770,50 @@
...
+    "copy-from": "book_fict_soft_satire_tpl"
+  },
+  {
+    "id": "book_fict_soft_satire_cats",
+    "type": "BOOK",
+    "name": "Cat’s Cradle",
+    "description": "A paperback edition of Kurt Vonnegut's fourth novel, in which the threat of nuclear destruction isn't much of an influence on human nature.",
+    "weight": "360 g",
+    "copy-from": "book_fict_soft_satire_tpl",
+    "relative": { "chapters": 111, "fun": -1, "time": -19 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:816]:
```suggestion
    "relative": { "chapters": 111, "fun": -1, "time": "-19 m" }
```

File: data/json/items/book/nonfiction.json [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+    "relative": { "chapters": -14 }
+  },
+  {
+    "id": "book_nonf_soft_wedding_coolring",
+    "type": "BOOK",
+    "name": "10 Cool Things About Being A Ring Bearer",
+    "//": "This book belongs to two genres: wedding and kids",
+    "description": "This book is for the delightful little ring bearer in your wedding.  The author depicts the responsibility and the honor in being a ring bearer your little angel will cherish.",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": -17, "intelligence": -2, "time": -5 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:77]:
```suggestion
    "relative": { "chapters": -17, "intelligence": -2, "time": "-5 m" }
```

File: data/json/items/book/nonfiction.json [hunk 14]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_parent_howtogen",
+    "type": "BOOK",
+    "name": "How to Raise a Gentleman: A Civilized Guide to Parenting",
+    "description": "A revised edition for parents who hope their little boy children will grow up to be the kind of men who know which fork to use, how to treat others, and generally make their parents proud.",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": -4, "time": -5 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:85]:
```suggestion
    "relative": { "chapters": -4, "time": "-5 m" }
```

File: data/json/items/book/nonfiction.json [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_safety_radiaea",
+    "type": "BOOK",
+    "name": "International Approaches to Securing Radioactive Sources Against Terrorism",
+    "description": "This book presents how to enhance cooperation and assistance between countries in support of International Atomic Energy Agency efforts to secure radioactive sources against the threat of terrorism.",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": 31, "time": -12 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:93]:
```suggestion
    "relative": { "chapters": 31, "time": "-12 m" }
```

File: data/json/items/book/nonfiction.json [hunk 16]
```diff
@@ -0,0 +1,207 @@
...
+  {
+    "id": "book_nonf_hard_psych_forpsych",
+    "type": "BOOK",
+    "name": "Principles of Forensic Psychiatry",
+    "description": "This text addresses standards in the assessment and treatment of aggression and violence as well as psychological and neuroimaging assessments.",
+    "weight": "2835 g",
+    "volume": "3673 ml",
+    "fun": -2,
+    "copy-from": "book_nonf_hard_tpl",
+    "relative": { "chapters": -9, "intelligence": 2, "time": 2 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:104]:
```suggestion
    "relative": { "chapters": -9, "intelligence": 2, "time": "2 m" }
```

File: data/json/items/book/nonfiction.json [hunk 17]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_hard_psych_grconres",
+    "type": "BOOK",
+    "name": "The Guide to Reflective Conflict Resolution",
+    "description": "The back cover of this hardbound book reads: \"Why should professionals care about reflective practice?  How do its principles and methods increase competence?  What characteristics distinguish reflective practitioners?\"",
+    "weight": "494 g",
+    "volume": "761 ml",
+    "copy-from": "book_nonf_hard_tpl",
+    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": 5 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:114]:
```suggestion
    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": "5 m" }
```

File: data/json/items/book/nonfiction.json [hunk 18]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_hard_psych_moodalm",
+    "type": "BOOK",
+    "name": "The Oxbridge Handbook of Mood Ailments",
+    "description": "The Oxbridge Handbook of Mood Ailments provides detailed coverage of the characterization, understanding, and treatment of affective disorders.  It provides coverage of unipolar depression, bipolar disorder, and known variants of these illnesses.",
+    "weight": "1152 g",
+    "volume": "1647 ml",
+    "copy-from": "book_nonf_hard_tpl",
+    "relative": { "chapters": 6, "fun": -1, "intelligence": 4, "time": 20 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:124]:
```suggestion
    "relative": { "chapters": 6, "fun": -1, "intelligence": 4, "time": "20 m" }
```

File: data/json/items/book/nonfiction.json [hunk 19]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_hard_psych_phonodis",
+    "type": "BOOK",
+    "name": "Phonological Acquisition and Disorders",
+    "description": "Studying the phonologies of children with non-organic speech disorders, this volume details the latest findings in optimality theory, phonological acquisition and disorders.  It is intended for linguists and psychologists.",
+    "weight": "730 g",
+    "volume": "1038 ml",
+    "copy-from": "book_nonf_hard_tpl",
+    "relative": { "chapters": -7, "fun": -2, "intelligence": 5, "time": 10 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:134]:
```suggestion
    "relative": { "chapters": -7, "fun": -2, "intelligence": 5, "time": "10 m" }
```

File: data/json/items/book/nonfiction.json [hunk 20]
```diff
@@ -0,0 +1,207 @@
...
+  {
+    "id": "book_nonf_hard_psych_thergar",
+    "type": "BOOK",
+    "name": "Therapeutic Gardens and Healing Spaces",
+    "//": "This is an example of book that belongs in two genres: homemk and psych",
+    "description": "This book addresses how to design therapeutic gardens.  It illustrates a variety of landscape designs appropriate for public spaces that promote mental health.",
+    "weight": "1043 g",
+    "volume": "1110 ml",
+    "copy-from": "book_nonf_hard_tpl",
+    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": -2 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:145]:
```suggestion
    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": "-2 m" }
```

File: data/json/items/book/nonfiction.json [hunk 21]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_psych_adrugds",
+    "type": "BOOK",
+    "name": "Advances in Drug Delivery Systems",
+    "description": "This softcover reprint covers an array of topics in pharmacology.  The physicochemical concepts of the refinement of bioresponsive drug delivery are presented in detail alongside a variety of current approaches employed in the development of zero order release systems.",
+    "weight": "773 g",
+    "volume": "964 ml",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": 2, "fun": -1, "intelligence": 5, "time": 10 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:155]:
```suggestion
    "relative": { "chapters": 2, "fun": -1, "intelligence": 5, "time": "10 m" }
```

File: data/json/items/book/nonfiction.json [hunk 22]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_psych_cggamers",
+    "type": "BOOK",
+    "name": "A Clinical Guide to Video Gamers",
+    "description": "This scholarly work considers the role that games play in psychological experiences and mental health.  Chapters examine the factors that compel individual gamers to select and identify with particular games and characters, as well as the different play styles, genres, and archetypes common in video games.",
+    "weight": "295 g",
+    "volume": "442 ml",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": -3, "fun": -2, "intelligence": 2, "time": 10 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:175]:
```suggestion
    "relative": { "chapters": -3, "fun": -2, "intelligence": 2, "time": "10 m" }
```

File: data/json/items/book/nonfiction.json [hunk 23]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_psych_paranoia",
+    "type": "BOOK",
+    "name": "Paranoia and the History of Madness",
+    "description": "This book is an analysis of the use and misuse of paranoia throughout history and in contemporary society.  The impact of paranoia on societies is explored in detail.",
+    "weight": "522 g",
+    "volume": "759 ml",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": 5 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:185]:
```suggestion
    "relative": { "chapters": -6, "fun": -1, "intelligence": 3, "time": "5 m" }
```

File: data/json/items/book/nonfiction.json [hunk 24]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_psych_psycolo",
+    "type": "BOOK",
+    "name": "Psychoanalysis and Colonialism",
+    "description": "Freud referred to women's sexuality as a \"dark continent\" for psychoanalysis, drawing on colonial use of the same phrase to refer to Africa.  This book details how the problematic universalism of psychoanalysis led theorists to reject its relevance for postcolonial critique.",
+    "weight": "454 g",
+    "volume": "734 ml",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "fun": -1, "intelligence": 3, "time": 10 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:195]:
```suggestion
    "relative": { "fun": -1, "intelligence": 3, "time": "10 m" }
```

File: data/json/items/book/nonfiction.json [hunk 25]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  {
+    "id": "book_nonf_soft_psych_psycstlk",
+    "type": "BOOK",
+    "name": "The Psychology of Stalking",
+    "description": "This book explores stalking from social, psychiatric, psychological and behavioral perspectives.  Topics include psychiatric diagnoses, offender-victim typologies, cyberstalking, false victimization syndrome, erotomania, domestic violence, the stalking of public figures, and many other aspects of stalking.",
+    "weight": "460 g",
+    "volume": "521 ml",
+    "copy-from": "book_nonf_soft_tpl",
+    "relative": { "chapters": 3, "intelligence": 2, "time": -2 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/nonfiction.json:205]:
```suggestion
    "relative": { "chapters": 3, "intelligence": 2, "time": "-2 m" }
```

File: data/json/items/book/dodge.json [hunk 26]
```diff
@@ -39,17 +58,21 @@
...
+    "type": "BOOK",
+    "name": { "str": "Treasury of Legends about Western Dancing", "str_pl": "copies of Western Dancing" },
+    "description": "Written by Emanuel Nogueira, a constabulario and historian of Nuevo Laredo, this massive book details the movements and cultural legacies of a variety of North American folk dances.",
+    "copy-from": "book_nonf_hard_dodge_tpl",
+    "weight": "2330 g",
+    "volume": "1250 ml",
+    "color": "blue",
+    "required_level": 4,
+    "max_level": 5,
+    "relative": { "price_post": 2000, "intelligence": 4, "time": 10 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/dodge.json:76]:
```suggestion
    "relative": { "price_post": 2000, "intelligence": 4, "time": "10 m" }
```

**Comment by Jerimee** [comment]
@ZhilkinSerg thanks for the edits - I wanted to use numerical rather than "20 m" because I could not get abstract inheritance to work with the string version.  If the abstract has "time: 20," the child can modify it via relative: -5 (or what have you).  That doesn't seem to be the case with the string notation.

Thanks in advance for your feedback.

**Comment by anothersimulacrum** [comment]
Could you make an issue about that? That's something that should be fixed.

**Comment by Jerimee** [comment]
> Could you make an issue about that? That's something that should be fixed.

Sure thing.  And I'll double check that it isn't a PEBCAK issue.

**Comment by Jerimee** [comment]
This PR has become unwieldy - can be merged as is?  

If not, I think I need to close it and then break it down into distinct parts so that it is easier to discuss, test, and merge.

The impact in-game will be that there are more clutter paperback books, particularly in those spawning from SUS itemgroups.

The bigger impact is the changes to the organization and taxonomy for book json and SUS library.  Hopefully what I've done here makes the desired organization clear.

**Comment by ZhilkinSerg** [comment]
We just need to fix some minor formatting issues reported in (double spaces and such). Can be handled while mergetesting.

File: data/json/items/book/abstract.json [hunk 27]
```diff
@@ -0,0 +1,238 @@
...
+    "time": "35 m",
+    "chapters": 30,
+    "//": "TODO fun should be conditional for this item",
+    "fun": 1
+  },
+  {
+    "abstract": "paperback_romance_spb",
+    "type": "BOOK",
+    "name": "Sweet Providence Romance Novel",
+    "description": "Sweet Providence Books is a publisher of discount romance paperbacks easily recognized by their blue and yellow cover illustrations. Despite the adult nature of their subject matter, the books tend to be under 250 pages of large print written in a vocabulary consistent with a 4th grade reading level.",
```
Comment (ZhilkinSerg) [inline data/json/items/book/abstract.json:126]:
```suggestion
    "description": "Sweet Providence Books is a publisher of discount romance paperbacks easily recognized by their blue and yellow cover illustrations.  Despite the adult nature of their subject matter, the books tend to be under 250 pages of large print written in a vocabulary consistent with a 4th grade reading level.",
```

File: data/json/items/book/misc.json [hunk 28]
```diff
@@ -823,18 +353,46 @@
...
     "intelligence": 6,
     "time": "18 m",
     "chapters": 24,
     "fun": 3
   },
+  {
+    "id": "book_fict_soft_pulp_squids",
+    "type": "BOOK",
+    "name": "Planet of the Murderous Squids that Time Forgot!",
+    "description": "In this psychedelic adventure novel of cosmic exploration, an elderly assassin discovers a planet too good to be true. Only once it is too late does she discover the harrowing truth at the center of \"The Planet of the Murderous Squids that Time Forgot!\"",
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json:366]:
```suggestion
    "description": "In this psychedelic adventure novel of cosmic exploration, an elderly assassin discovers a planet too good to be true.  Only once it is too late does she discover the harrowing truth at the center of \"The Planet of the Murderous Squids that Time Forgot!\"",
```

File: data/json/items/book/young.json [hunk 29]
```diff
@@ -0,0 +1,193 @@
...
+        "id": "fairyat_2025",
+        "text": "Titled \"The Fleeing Pancake\", this collection of silly folk tales is suitable for small children."
+      }
+    ]
+  },
+  {
+    "id": "book_fict_soft_ya_adrk",
+    "type": "BOOK",
+    "name": "The Adorkable Girl",
+    "description": "When a therapist's daughter transfers to a new school, she decides to change her personality type.  As her social life begins to blossom, can she maintain a healthy boundary between her home life and her public persona? ",
```
Comment (ZhilkinSerg) [inline data/json/items/book/young.json:133]:
```suggestion
    "description": "When a therapist's daughter transfers to a new school, she decides to change her personality type.  As her social life begins to blossom, can she maintain a healthy boundary between her home life and her public persona?",
```

File: data/json/items/book/abstract.json [hunk 30]
```diff
@@ -0,0 +1,238 @@
...
+    "symbol": "?",
+    "color": "pink",
+    "max_level": 1,
+    "intelligence": 5,
+    "time": "8 m",
+    "fun": 1
+  },
+  {
+    "abstract": "book_martial",
+    "type": "GENERIC",
```
Comment (ZhilkinSerg) [inline data/json/items/book/abstract.json:225]:
```suggestion
    "type": "BOOK",
```

File: data/json/items/book/abstract.json [hunk 31]
```diff
@@ -0,0 +1,238 @@
...
+    "category": "books",
+    "name": { "str": "martial art manual" },
+    "weight": "150 g",
+    "volume": "250 ml",
+    "price": 20000,
+    "material": "paper",
+    "symbol": "?",
+    "color": "green",
+    "use_action": "MA_MANUAL",
+    "time": "10 m",
```
Comment (ZhilkinSerg) [inline data/json/items/book/abstract.json:235]: `time` does not work for `GENERIC`, only for `BOOK` items:

```suggestion
    "time": "10 m",
```

File: data/json/items/book/abstract.json [hunk 32]
```diff
@@ -0,0 +1,237 @@
...
+    "category": "books",
+    "name": { "str": "martial art manual" },
+    "weight": "150 g",
+    "volume": "250 ml",
+    "price": 20000,
+    "material": "paper",
+    "symbol": "?",
+    "color": "green",
+    "use_action": "MA_MANUAL",
+    "book_data": { "intelligence": 9 }
```
Comment (ZhilkinSerg) [inline data/json/items/book/abstract.json:235]:
```suggestion
    "book_data": { "time": "10 m", "intelligence": 9 }
```

**Pull request merged**